Makefile

```diff
@@ -1,6 +1,7 @@
 DESTDIR :=
-BINDIR := /usr/local/bin
-SYSCONFDIR := /etc
+PREFIX := /usr/local
+BINDIR := $(PREFIX)/bin
```

Comment:
So that's where we have to make a decision. GNU things use /usr/local so a dev version won't override a system version. I've noticed we don't do that, not in cc 2.1, not in shim 3.0 either. I don't care that much but we need to do the same thing everywhere and propose the option to change that.

Installing the config from a dev version to a system one doesn't sound to good.